File: examples-sdk/go/205-llm-chunked-summarizer/main.go
// 205-llm-chunked-summarizer — Go SDK equivalent
// Compile: vil compile --from go --input 205-llm-chunked-summarizer/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	p := vil.NewPipeline("ChunkedSummarizerPipeline", 8080)

	p.Route("sink.trigger_out", "source_summarize.trigger_in", "LoanWrite")
	p.Route("source_summarize.response_data_out", "sink.response_data_in", "LoanWrite")
	p.Route("source_summarize.response_ctrl_out", "sink.response_ctrl_in", "Copy")

	p.Compile()
}

File: examples-sdk/java/205-llm-chunked-summarizer/Main.java
// 205-llm-chunked-summarizer — Java SDK equivalent
import dev.vil.*;
public class Main {
    public static void main(String[] args) {
        VilPipeline p = new VilPipeline("ChunkedSummarizerPipeline", 8080);
        p.route("sink.trigger_out", "source_summarize.trigger_in", "LoanWrite");
        p.route("source_summarize.response_data_out", "sink.response_data_in", "LoanWrite");
        p.route("source_summarize.response_ctrl_out", "sink.response_ctrl_in", "Copy");
        p.compile();
    }
}
